tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2558,6 +2802,37 @@ def test_whisper_longform_single_batch_prev_cond(self):
 
         assert decoded == EXPECTED_TEXT
 
+    @slow
+    def test_whisper_shortform_single_batch_prev_cond(self):
```

Comment:
would be nice to have both with and without here to see what this is acutally bringing to the table, because for all I know the expected text here is the same as without